### PR TITLE
Export QueueName type

### PR DESCRIPTION
**Pull request by BoussonKarel**
https://github.com/adopted-ember-addons/ember-file-upload/pull/890

### CODE REVIEW

File: ember-file-upload/src/index.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Queue } from './queue';
 import { UploadFile } from './upload-file';
 import FileQueueService from './services/file-queue';
 import { uploadHandler } from './mirage/upload-handler';
-import { FileSource, FileState } from './interfaces';
+import { FileSource, FileState, QueueName } from './interfaces';
```
Comment (gilest) [inline ember-file-upload/src/index.ts:5]: I think this should work.

Tested locally by linking a local build of `ember-file-upload` with this change into an test ember app with typescript.

```suggestion
import { FileSource, FileState, type QueueName } from './interfaces';
```

Comment (gilest) [reply ember-file-upload/src/index.ts:5]: @BoussonKarel Would you like to try this or shall I open a new PR?

**Comment by BoussonKarel** [comment]
Works and looks ok

**Review by gilest — APPROVED** [review]

**Comment by gilest** [comment]
Released in [7.3.0](https://github.com/adopted-ember-addons/ember-file-upload/releases/tag/v7.3.0). Thanks for your contribution!

**Pull request merged**